### ISSUE TITLE
Dilato: Core: There is no warning message appears on main brackets window after click "Restart" button on the one display the update options

### ISSUE THREAD

**Issue by ZuoBaogang** [adobe/brackets#14387]
Steps:

1. Open brackets 1.13
2. Create a new file and do some edit and don't save it
3. Go to Debug > New Brackets Window
4. Go to Help Menu in newly opened window
5. Click on Check for Updates --- Update Notification Dialogue should open in the same window where user clicked on Check for update.
6. Click on Get it Now button --- Download will start and you can see the download status in status bar
7. After download the update file, click restart --- the current Brackets windows will be colsed, Save Changes dialogue will appear on the main Brackets window
8. Click Cancel button --- there is no warning message appears
9. Save the file and close the Brackets --- It will update the brackets

Result: After step 8, there is no warning message appears

Expect: The warning message "The update will be applied on relaunch." should appears after step 8.

**Comment by ayushrathi15** [comment]
@RuchiAgrawalAdobe This is duplicate of #14375
